front/src/actions/message.js

```diff
@@ -74,11 +78,12 @@ function createActions(store) {
       });
       const messageText = state.currentMessageTextInput;
       try {
+        const tempId = uuid.v4();
         const newMessage = {
           text: messageText,
-          created_at: new Date()
+          created_at: new Date(),
+          uuid: tempId
```

Comment:
Why did you use a "uuid" attribute and not "id" like at most places?

Reply:
Like it was really an UUID (v4), I found more meanfull `uuid` than `id`. But it's fixed